pkg/apis/work/v1alpha2/well_known_labels.go

```diff
@@ -25,4 +25,15 @@ const (
 
 	// WorkNameLabel is added to objects to specify associated Work's name.
 	WorkNameLabel = "work.karmada.io/name"
+
+	// ResourceConflictResolutionAnnotation is added to the resource template to specify how to resolve the conflict
+	// in case of resource already existing in member clusters.
+	// The valid value is:
+	//   - overwrite: always overwrite the resource if already exist. The resource will be overwritten with the
+	//     configuration from control plane.
+	// Note: Propagation of the resource template without this annotation will fail in case of already exists.
+	ResourceConflictResolutionAnnotation = "work.karmada.io/conflict-resolution"
+
+	// ResourceConflictResolutionOverwrite is the value of ResourceConflictResolutionAnnotation, indicating the overwrite strategy.
+	ResourceConflictResolutionOverwrite = "overwrite"
```

Comment:
I guess we should not put a "value" here, maybe put it in `pkg\util\constants.go`?

Reply:
It's part of the API, I don't think we should separate them.

Reply:
> It's part of the API, I don't think we should separate them.

Yes, but do we need to put these values in a separate file in the API directory?

Reply:
But we can group the resolution by a separated `const`:
```golang
// Define resource conflict resolution
const (
	// ResourceConflictResolutionAnnotation is added to the resource template to specify how to resolve the conflict
	// in case of resource already existing in member clusters.
	// The valid value is:
	//   - overwrite: always overwrite the resource if already exist. The resource will be overwritten with the
	//     configuration from control plane.
	// Note: Propagation of the resource template without this annotation will fail in case of already exists.
	ResourceConflictResolutionAnnotation = "work.karmada.io/conflict-resolution"

	// ResourceConflictResolutionOverwrite is the value of ResourceConflictResolutionAnnotation, indicating the overwrite strategy.
	ResourceConflictResolutionOverwrite = "overwrite"
)
```

Reply:
Done, PTAL